internal/update/realtime/realtime_test.go

```diff
@@ -204,12 +210,520 @@ func TestUpdateTrips(t *testing.T) {
 	}
 }
 
+func TestUpdateVehicles(t *testing.T) {
```

Comment:
Thanks for adding so many unit tests!

Reply:
No problem, was relatively easy with the test infrastructure you already had in place!